turbinia/workers/volatility.py

```diff
@@ -26,11 +26,7 @@
 
 
 class VolatilityTask(TurbiniaTask):
-  """Task to execute volatility.
-
-  Attributes:
-    module(str): The name of the volatility module to run.
-  """
+  """Task to execute volatility."""
```

Comment:
Why did the module get removed from the docstring?  It's still an attribute of this object.

Reply:
Good catch, I changed that.